simde/arm/neon/create.h

```diff
@@ -23,11 +23,13 @@
  * Copyright:
  *   2020      Evan Nemerson <evan@nemerson.com>
  *   2020      Christopher Moore <moore@free.fr>
+ *   2023      Yi-Yen Chung <eric681@andestech.com> (Copyright owned by Andes Technology)
  */
 
 /* N.B. CM: vcreate_f16 and vcreate_bf16 are omitted as
  * SIMDe has no 16-bit floating point support.
  * Idem for the poly types. */
```

Comment:
SIMDe has 16-bit floating point support now; the lack of poly types isn't relevant here (I think)
```suggestion
```

Reply:
Alright, I will remove it.